tests/saitests/py3/sai_qos_tests.py

```diff
@@ -2720,6 +2720,12 @@ def runTest(self):
         assert (len(sidx_dscp_pg_tuples) >= self.pgs_num)
         print(sidx_dscp_pg_tuples, file=sys.stderr)
         sys.stderr.flush()
+        # Collect destination ports that may be in a lag
+        pkt_dst_mac = self.router_mac if self.router_mac != '' else self.dst_port_mac
+        dst_port_ids = []
+        for i in range(len(self.src_port_ids)):
```

Comment:
Can you please clarify why dst ports and dependent on src_port_id ? Should we check if destination port is part of the lag then disable all ports under that lag ?